src/WSOverrideLayoutBuilder.php

```diff
@@ -134,7 +134,7 @@ private function getSelectedColorway(EntityInterface $entity): string {
     $stored_ws_styles = $entity?->override_styles?->value
       ? unserialize($entity?->styles?->value)
       : $this->getDefaultWsStyle($entity->bundle());
-    return $stored_ws_styles['colorway'];
+    return $stored_ws_styles['colorway'] ?? '';
```

Comment:
To fix
![image](https://github.com/user-attachments/assets/7da704b9-c084-4ca9-80b0-db6a07085a36)